backend/app/controllers/spree/admin/images_controller.rb

```diff
@@ -13,7 +13,7 @@ class ImagesController < ResourceController
       private
 
       def location_after_destroy
-        spree.admin_product_images_url(@arent)
```

Comment:
@MatthewKennedy can we add this to controller spec?

Reply:
Sure.